engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -95,6 +99,8 @@ object WorkflowExecutionActor {
     override val exceptionContext = s"WorkflowExecutionActor"
   }
 
+  val MaxRetries = ConfigFactory.load().getInt("system.maxRetries")
```

Comment:
Currently -- the use case for this functionality is via the WDL "preemptible" count... is that what we are handling here?